specification/subscription/resource-manager/Microsoft.Subscription/preview/2024-08-01-preview/subscriptions.json

```diff
@@ -753,6 +753,141 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Subscription/transfers/{transferId}": {
+      "put": {
+        "description": "Transfers subscription billing ownership to the destination email. The TransferId provided in the PUT request is client-generated for tracking purposes and is different from the system-generated TransferId returned in subsequent responses.",
+        "operationId": "Transfers_SendExport",
+        "x-ms-examples": {
+          "CreateTransfer": {
+            "$ref": "./examples/createTransfer.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Transfer creation request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TransferCreationRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Initiated Transfer details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "description": "Retrieves transfer details using a system-generated TransferId. ",
+        "operationId": "Transfers_GetById",
+        "x-ms-examples": {
+          "GetTransferById": {
+            "$ref": "./examples/getTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Transfer details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "description": "Cancels the transfer using the system-generated TransferId.",
+        "operationId": "Transfers_CancelById",
+        "x-ms-examples": {
+          "GetTransferById": {
+            "$ref": "./examples/cancelTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
```

Comment:
>responses [](http://example.com/codeflow?start=9&length=9)

Please refer [https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/delete-resource.md](https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/delete-resource.md)

and have 200 and 204 accordingly. Based on your scenario 200 is also possible.